src/String.cc

```diff
@@ -163,8 +163,8 @@ String::append( char const *str, int len)
     assert(str && len >= 0);
 
     PROF_start(StringAppend);
-    if (len_ + len < size_) {
-        strncat(buf_, str, len);
+    if (len_ + len + 1/*'\0'*/ < size_) {
```

Comment:
Please put a space after `1` so that the following `/` groups more with its `*` and looks less like a division operator. I do not insist on this change.

Reply:
done